cfme/tests/automate/test_git_import.py

```diff
@@ -79,27 +79,29 @@ def test_automate_git_domain_displayed_in_service(appliance, imported_domain):
         initialEstimate: 1/20h
         tags: automate
     """
+    url = "https://github.com/ramrexx/CloudForms_Essentials.git"
```

Comment:
**REQUIRED** : please move this to Yamls. Link to yamls shared in gchat.

Reply:
@kedark3 
I am agree with you but this URL or git repo may vary if I want to change or update this test case
So for now I am forking this repo on my GitHub account and adding url of forked repo in Yamls.
PR: https://gitlab.cee.redhat.com/cfme-qe/cfme-qe-yamls/merge_requests/811

Reply:
I have merged that PR.